test/resources/reftest-analyzer.html

```diff
@@ -103,73 +103,76 @@ <h1>Reftest analyzer</h1>
       <label>
         <input type="checkbox" id="differences"> Circle differences
       </label>
+      <span id="shortcuts">Shortcuts: n=next p=previous t=toggle</span>
```

Comment:
Note that you also have <kbd>d</kbd>, which toggles the "Circle differences" feature.